feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenViewModel.kt

```diff
@@ -122,6 +115,32 @@ class SearchScreenViewModel @Inject constructor(
         )
     }
 
+    fun onSessionTypesSelected(sessionType: TimetableSessionType, isSelected: Boolean) {
+        val selectedSessionTypes = searchFilterUiState.value.selectedSessionTypes.toMutableList()
+        searchFilterUiState.value = searchFilterUiState.value.copy(
+            selectedSessionTypes = selectedSessionTypes.apply {
+                if (isSelected) {
+                    add(sessionType)
+                } else {
+                    remove(sessionType)
+                }
+            },
+        )
+    }
+
+    fun onLanguagesSelected(language: Lang, isSelected: Boolean) {
+        val selectedLanguages = searchFilterUiState.value.selectedLanguages.toMutableList()
+        searchFilterUiState.value = searchFilterUiState.value.copy(
```

Comment:
I don't think we should copy the UiState because this broke the single source of UiState 💭 
But this is existence implementation so I'm going to create issue and it is ok to leave it as it is 👍 
https://github.com/DroidKaigi/conference-app-2023#single-source-of-truth-with-builduistate-